ui/app/src/services/contentTypes.ts

```diff
@@ -71,7 +89,29 @@ function getFieldValidations(
 ): Partial<ContentTypeFieldValidations> {
   const map = asArray<LegacyFormDefinitionProperty>(fieldProperty).reduce<LookupTable<LegacyFormDefinitionProperty>>(
     (table, prop) => {
-      table[prop.name] = prop;
+      if (prop.name === 'width' || prop.name === 'height') {
+        const parsedValidation = JSON.parse(prop.value);
+        if (parsedValidation.exact) {
+          table[prop.name] = {
+            name: prop.name,
+            type: prop.type,
+            value: parsedValidation.exact
+          };
+        } else {
+          table[`min${capitalize(prop.name)}`] = {
+            name: prop.name,
+            type: prop.type,
+            value: parsedValidation.min
+          };
+          table[`max${capitalize(prop.name)}`] = {
+            name: prop.name,
+            type: prop.type,
+            value: parsedValidation.max
+          };
+        }
```

Comment:
@rart just for you to know, values are set unparsed in here, but later in code values are parsed before returning the validation fields.

Reply:
- Where about?
- Why wait/distribute?